distribution/docker/build.gradle

```diff
@@ -303,8 +310,10 @@ void addBuildDockerImageTask(Architecture architecture, DockerBase base) {
       onlyIf { Architecture.current() == architecture }
     }
 
-  tasks.named("assemble").configure {
-    dependsOn(buildDockerImageTask)
+  if (base != DockerBase.IRON_BANK) {
```

Comment:
Won't this break release manager since now running `assemble` won't produce this artifact?

Reply:
Shouldn't do - release-manager only cares about the Docker context, not the image.

Reply:
Oh I see. Got it.